docs/eventlisteners.md

```diff
@@ -98,8 +98,11 @@ The `triggers` field is required. Each EventListener can consist of one or more
 - `name` - (Optional) a valid
   [Kubernetes name](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#syntax-and-character-set)
 - [`interceptors`](#interceptors) - (Optional) list of interceptors to use
-- `bindings` - A list of `TriggerBindings` reference to use or embedded TriggerBindingsSpecs to use.
-- `template` - The name of `TriggerTemplate` to use
+- `bindings` - (Optional) A list of `TriggerBindings` reference to use or embedded TriggerBindingsSpecs to use.
+- `template` - (Optional) The name of `TriggerTemplate` to use
+- `triggerRef` - (Optional) Reference to the [`Trigger`](./trigger.md).
+
+A `trigger` field must have combination of `bindings`, `template` and `interceptors` or `triggerRef`.
```

Comment:
Can we provide format on how we can use `triggerRef` as part of `trigger`  because right now we have trigger with `bindings`, `template` and `interceptors`

Reply:
Done that.